src/components/pages/InstitutionVoucher.vue

```diff
@@ -506,7 +506,7 @@ export default {
     subjectRules() {
       return [
         // eslint-disable-next-line no-control-regex
-        (v) => (this.validate(v)) || 'Beschreibung muss Base64 encoded sein',
+        (v) => (this.validate(this.window.btoa(v))) || 'Beschreibung muss Base64 encoded sein',
```

Comment:
confusion 100

Reply:
Mit window.btoa wird das in base 64 kodiert. In der validate Funktion wird das dekodiert und falls ne exception auftritt ist es nicht base 64

Reply:
Sollte es dann nicht besser "Beschreibung muss Base64 encodedierbar sein" heißen?
Der eigentliche codec is bei javascript meine ich auch latin-1

Reply:
Jo haste recht. Änder ich gleich